Client/SafeBrowsingPages/InterstitialPage.swift

```diff
@@ -0,0 +1,214 @@
+                "transparency_report": "https://transparencyreport.google.com/safe-browsing/search?url=\(escapedURL)",
+                "detected_problem": "detected phishing",
+                "error_detection_2": "on \(host).\nPhishing sites pretend to be other websites to trick you.",
+                "report_problem": "You can",
+                "report_problem_url": "https://safebrowsing.google.com/safebrowsing/report_error/?url=\(escapedURL)",
+                "report_problem_text": "report a detection problem",
+                "risk_1": "or, if you understand the risks to your security,",
+                "risk_2": "visit this unsafe site"
+            ]
+            return buildResponse(asset: asset, variables: variables)
+            
+        case .unwantedSoftware:
+            let asset = Bundle.main.path(forResource: "UnwantedSoftware", ofType: "html")
+            let variables = [
+                "tab_title": "Safe Browsing",
+                "page_header": "Harmful site ahead",
+                "error_description": "Attackers on \(host) might attempt to trick you into installing programs that harm your browsing experience (for example, by changing your homepage or showing extra ads on sites you visit).",
+                "learn_more": "Learn more",
+                "more_details": "More details",
+                "back_to_safety": "Back to safety",
+                "error_detection": "Google Safe Browsing recently detected phishing on \(host).\nPhishing sites pretend to be other websites to trick you.",
+                "report_problem": "You can",
+                "report_problem_url": "https://safebrowsing.google.com/safebrowsing/report_error/?url=\(escapedURL)",
+                "report_problem_text": "report a detection problem",
+                "risk_1": "or, if you understand the risks to your security,",
+                "risk_2": "visit this unsafe site"
+            ]
+            return buildResponse(asset: asset, variables: variables)
+            
+        case .potentiallyHarmfulApplication:
+            let asset = Bundle.main.path(forResource: "HarmfulApplication", ofType: "html")
+            let variables = [
+                "tab_title": "Safe Browsing",
+                "page_header": "Harmful site ahead",
+                "error_description": "Attackers on \(host) might attempt to trick you into installing programs that harm your browsing experience (for example, by changing your homepage or showing extra ads on sites you visit).",
+                "learn_more": "Learn more",
+                "more_details": "More details",
+                "back_to_safety": "Back to safety",
+                "error_detection": "Google Safe Browsing recently detected phishing on \(host).\nPhishing sites pretend to be other websites to trick you.",
+                "report_problem": "You can",
+                "report_problem_url": "https://safebrowsing.google.com/safebrowsing/report_error/?url=\(escapedURL)",
```

Comment:
Again, this report error page is only for phishing. There shouldn't be a link for potentially harmful applications.